### DISCUSSION TITLE
Purge css not working

### DISCUSSION THREAD

**Discussion by razvanstatescu** [tailwindlabs/tailwindcss#3084] [Help]
Hi, so I have an Angular multi project app. I'm using Tailwind with postcss and this is my tailwind config:
`
module.exports = {
  purge: {
    enabled: true,
    content: [
      './projects/provider/src/**/*.html',
      './projects/provider/src/**/*.ts',
    ]
  },
  theme: {},
  variants: {},
  plugins: [],
};
`
When I run NODE_ENV=production ng build --project=provider I get  Tailwind is not purging unused styles because no template paths have been provided.

Thanks!

**Comment by simonswiss** [comment]
Hey!

Could you perhaps share the repo so it's easier to help troubleshoot?

Do you by any chance also have the purge stuff defined in your PostCSS config file? If so, remove that and only configure it within your Tailwind config file.
